content/about/Acknowledgements/acknowledgements.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
```

Comment:
This file is in the "Acknowledgements" category, which is capitalized, but I think the directory shouldn't be capitalized.

Reply:
fixed